scripts/build-homepage

```diff
@@ -20,8 +20,9 @@ kdb --version
 # prevent jenkins from killing daemons
 BUILD_ID=dontKillMe
 
-# then restart the backend
+# then restart the backend (sleep prevents 'address already in use')
 kdb stop-rest-backend
+sleep 3
```

Comment:
You might be able to avoid this using a different `service.backlog` and search for SO_REUSEADDR. We really should avoid the backend to be down for every rebuild. What about trying to start it immediately, but restart if the address was in use?